pkg/services/annotations/annotationsimpl/xorm_store.go

```diff
@@ -377,41 +385,30 @@ func (r *xormRepositoryImpl) Get(ctx context.Context, query annotations.ItemQuer
 	return items, err
 }
 
-func (r *xormRepositoryImpl) getAccessControlFilter(user identity.Requester, accessResources *accesscontrol.AccessResources) (string, error) {
+func (r *xormRepositoryImpl) getAccessControlFilter(user identity.Requester, accessResources *accesscontrol.AccessResources) (string, []any, error) {
 	if accessResources.SkipAccessControlFilter {
-		return "", nil
+		return "", nil, nil
 	}
 
 	var filters []string
+	var params []any
```

Comment:
Just a sanity check – is there a reason this is not `[]string`?

Reply:
Yeah, the params it appends to in the parent function is `[]interface`: https://github.com/grafana/grafana/pull/106676/files#diff-5897ac24653e3393d73572ed7852f722660f6e9cee73dcd5f7b6b73ce8584f21L250

Reply:
Ah, right, now it makes sense. Thank you!